### PR TITLE
Table: Reduce empty space in the blueprint list

### LINKED ISSUE #1686: Add more information and columns for the blueprint table

### PR DESCRIPTION
**Pull request by regexowl**
Fixes #1686.

This reduces the amount of the empty space in the list of the blueprints.

### CODE REVIEW

**Comment by jkozol** [comment]
@regexowl I added additional info to the issue

**Comment by regexowl** [comment]
@jkozol Updated according to the additional info

File: src/components/Table/BlueprintTable.js [hunk 1]
```diff
@@ -2,24 +2,71 @@ import React from "react";
...
               <Link to={`/${blueprint.name}`}>{blueprint.name}</Link>
             </Td>
+            <Td>{blueprint.version}</Td>
+            <Td>{getLatestCreationDate(props.images, blueprint)}</Td>
+            <Td>
+              {
+                props.images.filter(
+                  (image) => image.blueprint === blueprint.name
+                ).length
+              }
```
Comment (jkozol) [inline src/components/Table/BlueprintTable.js:67]: Can you move this to a seperate function like `getLatestCreationDate`

File: src/components/Table/BlueprintTable.js [hunk 2]
```diff
@@ -2,24 +2,71 @@ import React from "react";
...
+    return "-";
+  }
+};
+
+const timestampToDate = (timestamp) => {
+  var options = { year: "numeric", month: "short", day: "numeric" };
+  var date = new Date(timestamp * 1000);
+  return date.toLocaleDateString("en-US", options);
+};
+
```
Comment (jkozol) [inline src/components/Table/BlueprintTable.js:36]: In the `helpers.js` file we declare a `formTimestampLabel` function that is used in the images table. Can you call that here to keep the label consistent.

**Review by jkozol — CHANGES_REQUESTED** [review]
This looks good :) Just a couple nitpicks.

File: src/components/Table/BlueprintTable.js (hunk 2 above)
Comment (regexowl) [reply src/components/Table/BlueprintTable.js]: Oops, didn't notice that one. Fixed!

File: src/components/Table/BlueprintTable.js (hunk 1 above)
Comment (regexowl) [reply src/components/Table/BlueprintTable.js]: Moved :+1:

**Review by jkozol — APPROVED** [review]
:)

**Pull request merged**